### PR TITLE
Fixed #9649 - Spanish Translations corrected

### LINKED ISSUE #9649: Incomplete Spanish translations
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

2.1, master

### Issue Description

The Bagisto Installer, Shop and Admin have incomplete/incorrect Spanish translations. Sometimes the translations are in English or German. As a native Spanish speaker I can create a PR with the fixes.

### Preconditions

N/A

### Steps to reproduce

Change to Spanish Language.

### Expected Result

The expected result is to have all correct Spanish translations.

### Actual Result

incomplete/incorrect Spanish translations

### Screenshots

Installer:
installer
Shop:
Shop
Admin
admin

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by dupotey**
## Issue Reference
#9649

## Description
The Spanish translations for the Bagisto Installer, the Shop and the Admin has been corrected.

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@dupotey 

Thank you so much for your contribution to Bagisto and we really appreciate your efforts, We have checked and PR and it seems alright and working fine. 
Please check the screenshot below.

Screenshot from 2024-04-12 13-59-24
Screenshot from 2024-04-12 13-59-51
Screenshot from 2024-04-12 14-00-02
Screenshot from 2024-04-12 14-00-27

File: packages/Webkul/Admin/src/Resources/lang/es/app.php [hunk 1]
```diff
@@ -1495,8 +1495,8 @@
 
             'campaigns' => [
                 'index' => [
-                    'create-btn' => 'Create Campaign',
-                    'title'      => 'Campaigns',
+                    'create-btn' => 'Crear Campaña',
+                    'title'      => 'Campañas',
 
                     'datagrid' => [
                         'active'   => 'Active',
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/lang/es/app.php:1502]: Hello @dupotey ,
Can you please add this datagrid key translation as well? 

The datagrid key is not translated into Spanish.

File: packages/Webkul/Installer/src/Resources/lang/es/app.php [hunk 2]
```diff
@@ -184,136 +184,136 @@
...
+                    'name'    => 'Mejores Colecciones',
                 ],
             ],
         ],
 
         'user'      => [
             'roles' => [
-                'description' => 'This role users will have all the access',
+                'description' => 'Los usuarios con este rol tendrán acceso a todo',
                 'name'        => 'Administrator',
```
Comment (amit-webkul) [inline packages/Webkul/Installer/src/Resources/lang/es/app.php:312]: This is also missing in Spanish.

It should be **Administradora** in Spanish.

**Review by amit-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/lang/es/app.php [hunk 3]
```diff
@@ -1511,122 +1511,122 @@
                 ],
 
                 'create'    => [
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/es/app.php:1513]: Please Remove extra space before double arrow there should be only one space. 
For example:
'create' => [
],

**Comment by jitendra-webkul** [comment]
@dupotey, thank you for your contribution, it's greatly appreciated!

**Pull request merged**